Description: Patch to enable translation support
 This patch sets the translation domain in the Gtk::Builder object
 used in the file src/gnome-media-player.cc . This sets the application
 GUI to use the available translations. This patch has been forwarded
 upstream and has been accepted for the next upstream release.
Author: Bilal Akhtar <bilalakhtar96@yahoo.com>
--- a/src/gnome-media-player.cc
+++ b/src/gnome-media-player.cc
@@ -264,7 +264,7 @@
 		{
 			builder = Gtk::Builder::create_from_file(UI_FILE);
 			ui_manager = Gtk::UIManager::create();
-
+			builder->set_translation_domain(GETTEXT_PACKAGE);
 			Gtk::RadioButtonGroup	engine_group;
 
 			action_about			= Gtk::Action::create("About", Gtk::Stock::ABOUT);
